content/guides/basic-setup/index.md

```diff
@@ -251,8 +251,30 @@ PS2 game discs are unencrypted DVDs and CDs. So, they can be dumped quickly usin
 
 ### How it works
 
-- Install ImgBurn
-- Put your PS2 game disc into a DVD drive
+- Install and run ImgBurn
+- Put your game disc into an optical drive
 - Create an image file from a disc inside ImgBurn (highlighted in screenshot below)
 
-{{< img src="./img/imgburn.webp" cols=6 >}}
+![ImgBurn](./img/imgburn.webp)
+
+## Dumping PlayStation 2 discs with Media Preservation Frontend (more advanced)
```

Comment:
Officially we recommend imgburn, no issue with some alternative methods being added but can we label this one as `Alternative: Dumping ....`?

Also it looks like the tool is windows only, let's add that somewhere.  This kinda just made me realize we have no image dumping instructions for linux users as well!

Reply:
Sorry for the late response, I forgot that I sent this PR and then the reply got lost in my notifications. 😩 

Yeah, I can do that for sure! Linux support for this tool is in development, so at some point we might be able to reuse that info for Linux and Mac as well.